.github/workflows/playwright.yml

```diff
@@ -0,0 +1,30 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  test:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Install dependencies
+      run: yarn
+    - name: Install Playwright Browsers
+      run: yarn playwright:install
+    - name: Run Playwright tests
+      run: yarn test
+      env:
+        # VITE_PROJECT_ID: ${{ secrets.VITE_PROJECT_ID }}
+        VITE_PROJECT_ID: 547aafa48826c4d76f492efecde4843d
```

Comment:
what's the game plan here?

Reply:
Will [update the secret](https://walletconnect.slack.com/archives/C03RME3BS9L/p1702856517094549) and then force push to erase this. Pushed temporarily in public since it isn't critically private info, but this really is a secret/variable.